### PR TITLE
Create match unit test.

### PR DESCRIPTION
**Pull request by lyw0149**
1. Create match unit test.
2. Use latest mongodb when test with travis.

@nlpenguin @ethkim @ace0625 @SarahhhC @KaylaKwon @djflexible  @heartjigi

### CODE REVIEW

**Comment by heartjigi** [comment]
:squirrel:

**Comment by heartjigi** [comment]
:squirrel:

**Comment by heartjigi** [comment]
:shipit:

File: src/test/controllers/match.controller.test.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+ * Test for User API
+ */
+
+const API_BASE_URL = 'http://localhost:8000';
+
+let matchData;
+
+describe('Test Match API', function () {
+
+  describe('User B request mentoring to User A.', () => {
```
Comment (ochanje210) [inline src/test/controllers/match.controller.test.js]: 'Requests'

File: src/test/controllers/match.controller.test.js [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+          should.fail();
+          done();
+        })
+        .catch(function (err) {
+          err.statusCode.should.equal(400);
+          done();
+        });
+    });
+
+    it('User B request mentoring to User A', function (done) {
```
Comment (ochanje210) [inline src/test/controllers/match.controller.test.js]: 'Requests'

**Comment by ochanje210** [comment]
U want me to delete the branch?

**Pull request merged**